strawberryfields/circuitspecs/chip2.py

```diff
@@ -0,0 +1,236 @@
+# Copyright 2019-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Circuit class specification for the chip0 class of circuits."""
```

Comment:
Looks like this should be `chip2` here :slightly_smiling_face:

Reply:
```suggestion
"""Circuit class specification for the chip2 class of circuits."""
```